DEV_START_GUIDE.md

```diff
@@ -0,0 +1,129 @@
+# Developer Quick Start Guide
+
+---
+
+CodeCharta consists of two separate parts:
+
+-   The [analysis](https://maibornwolff.github.io/codecharta/docs/analysis/) which is a cli-tool that generates a `.cc.json file` file.
+-   The [visualization](https://maibornwolff.github.io/codecharta/docs/visualization/) that consumes said file and visualises it in form of a tree map. The visualization has both a desktop client and a [web version]({{site.web_visualization_link}}).
+
+Both parts are in active development, meaning as a developer you can contribute to both.
+
+# Requirements
+
+To work on CodeCharta, please ensure your system includes:
+
+-   Git
+-   Java >= 11
+-   Node >= 16
+
+# Install guide
+
+To start contributing to codecharta, first clone the [GitHub repository](https://github.com/MaibornWolff/codecharta) and navigate into it
+
+```bash
+# clone the repository
+git clone https://github.com/MaibornWolff/codecharta.git
```

Comment:
Sorry for being too late. Please use the ssh remote url instead. @fritschldwg would you please adjust it?